src/nwb2bids/sanitization/_sanitization.py

```diff
@@ -0,0 +1,56 @@
+import pathlib
+import typing
+
+import pydantic
+
+from ._common import _sanitize_label
+from ._levels import SanitizationLevel
+
+
+class Sanitization(pydantic.BaseModel):
+    """
+    Sanitize all relevant metadata fields when translating from minimal NWB towards valid BIDS.
+
+    sanitization_level : SanitizationLevel
+        The level of sanitization to apply.
+        See `nwb2bids.sanitization.SanitizationLevel?` for more details.
+    sanitization_file_path : pathlib.Path
+        The file to save a record of sanitization actions taken.
+    original_session_id : str
+        The original session ID to be sanitized.
+    original_participant_id : str
+        The original participant ID to be sanitized.
+    sanitized_session_id : str
+        The sanitized session ID label.
+    sanitized_participant_id : str
+        The sanitized participant ID label.
+    """
+
+    sanitization_level: SanitizationLevel
+    sanitization_file_path: pathlib.Path
+    original_session_id: str
+    original_participant_id: str | None = None
+    sanitized_session_id: str | None = None
+    sanitized_participant_id: str | None = None
```

Comment:
It is more appropriate to make these two values into properties or computed fields because they are in fact computed. 

The current definition
1. makes the intent of these two values less clear
2. the values of `sanitized_session_id` and `sanitized_participant_id` can get out of sync with `original_session_id` and `sanitized_participant_id` respectively
3. The JSON Schema for the model for the validation mode will include these two values.